integrated_channels/canvas/client.py

```diff
@@ -81,7 +83,24 @@ def _post(self, url, data):
         response = self.session.post(url, data=data)
         return response.status_code, response.text
 
```

Comment:
does pylint ask for two lines here or one? I forget. I see above have two lines ... be good to be consistent I guess

Reply:
Where are you seeing the double line? I thought pylint required it for when you define classes, class methods seem to be single lined unless I'm missing something?

Reply:
guess I was dreaming lol! sorry